src/dns/aardvark.rs

```diff
@@ -101,19 +100,18 @@ impl Aardvark {
 
         log::debug!("start aardvark-dns: {:?}", aardvark_args);
 
-        let output = match log_enabled!(log::Level::Debug) {
-            true => Stdio::inherit(),
-            false => Stdio::null(),
-        };
-
+        // After https://github.com/containers/aardvark-dns/pull/148 this command
+        // will block till aardvark-dns's parent process returns back and let
+        // aardvark inherit all the fds.
         Command::new(&aardvark_args[0])
             .args(&aardvark_args[1..])
-            .stdin(Stdio::null())
-            .stdout(Stdio::null())
-            .stderr(output)
+            .stdin(Stdio::inherit())
+            .stdout(Stdio::inherit())
+            .stderr(Stdio::inherit())
             // set RUST_LOG for aardvark
             .env("RUST_LOG", log::max_level().as_str())
-            .spawn()?;
+            .output()
+            .expect("failed to start aardvark-dns process");
```

Comment:
do not panic! please return a proper error

Reply:
I think `?` will return error from the function. Did you mean wrapping the error.

Reply:
fixed.